### ISSUE TITLE
surface wireframes on by default, won't turn off

### ISSUE THREAD

**Issue by sofroniewn** [napari/napari#3878]
## 🐛 Bug
When I add a new surface I always see the wireframes, even if i try turning them off. It says they should not be visible according to the model code, but i still see them render. for example 

![image](https://user-images.githubusercontent.com/6531703/147325130-2d49c8a4-7fdf-4f2e-975d-07f85d3ff53c.png)

currently on main, with latest vispy. cc @kevinyamauchi 

we should fix before release

**Comment by sofroniewn** [comment]
cc @brisvag @alisterburt

**Comment by sofroniewn** [comment]
I believe the bug is here https://github.com/napari/napari/blob/140feaa9a2e18c60f32eec96faa5945f7aa3b16b/napari/_vispy/layers/surface.py#L120

This should be
```
self.node.wireframe_filter.enabled = self.layer.wireframe.visible
```

**Comment by brisvag** [comment]
Yup, that's a leftover from pre-evented model I think!

**Comment by kevinyamauchi** [comment]
I believe this was fixed by #3879, so I am closing this issue. Please feel free to re-open if you find this wasn't fixed.
